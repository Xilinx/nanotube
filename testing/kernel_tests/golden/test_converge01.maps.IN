###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
nanotube_map: 0 0 3 8
key: 66 6f 6f
value: 00 00 00 00 00 00 00 00
key: 63 6f 64
value: 01 01 01 01 01 01 01 01
key: 68 61 74
value: 02 02 02 02 02 02 02 02
key: 63 61 62
value: 03 03 03 03 03 03 03 03
end

nanotube_map: 1 0 3 8
key: 62 61 72
value: 04 04 04 04 04 04 04 04
key: 77 69 70
value: 05 05 05 05 05 05 05 05
key: 63 61 74
value: 06 06 06 06 06 06 06 06
end

nanotube_map: 2 0 3 64
end

nanotube_map: 3 0 3 8
key: 63 61 74
value: 07 07 07 07 07 07 07 07
end
