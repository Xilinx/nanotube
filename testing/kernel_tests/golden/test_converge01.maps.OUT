###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
# Testing kernel cycle_switch_map_test
# Maps after packet 0
nanotube_map: 0 0 3 8
key: 63 61 62
value: 03 03 03 03 03 03 03 03
key: 68 61 74
value: 02 02 02 02 02 02 02 02
key: 63 6f 64
value: 01 01 01 01 01 01 01 01
key: 66 6f 6f
value: 00 00 00 00 00 00 00 00
end

nanotube_map: 1 0 3 8
key: 63 61 74
value: 06 06 06 06 06 06 06 06
key: 77 69 70
value: 05 05 05 05 05 05 05 05
key: 62 61 72
value: 04 04 04 04 04 04 04 04
end

nanotube_map: 2 0 3 64
key: 72 65 73
value: 00 04 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
end

nanotube_map: 3 0 3 8
key: 63 61 74
value: 07 07 07 07 07 07 07 07
end

# Maps after packet 1
nanotube_map: 0 0 3 8
key: 63 61 62
value: 03 03 03 03 03 03 03 03
key: 68 61 74
value: 02 02 02 02 02 02 02 02
key: 63 6f 64
value: 01 01 01 01 01 01 01 01
key: 66 6f 6f
value: 00 00 00 00 00 00 00 00
end

nanotube_map: 1 0 3 8
key: 63 61 74
value: 06 06 06 06 06 06 06 06
key: 77 69 70
value: 05 05 05 05 05 05 05 05
key: 62 61 72
value: 04 04 04 04 04 04 04 04
end

nanotube_map: 2 0 3 64
key: 72 65 73
value: 01 05 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
end

nanotube_map: 3 0 3 8
key: 63 61 74
value: 07 07 07 07 07 07 07 07
end

# Maps after packet 2
nanotube_map: 0 0 3 8
key: 63 61 62
value: 03 03 03 03 03 03 03 03
key: 68 61 74
value: 02 02 02 02 02 02 02 02
key: 63 6f 64
value: 01 01 01 01 01 01 01 01
key: 66 6f 6f
value: 00 00 00 00 00 00 00 00
end

nanotube_map: 1 0 3 8
key: 63 61 74
value: 06 06 06 06 06 06 06 06
key: 77 69 70
value: 05 05 05 05 05 05 05 05
key: 62 61 72
value: 04 04 04 04 04 04 04 04
end

nanotube_map: 2 0 3 64
key: 72 65 73
value: 02 06 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
end

nanotube_map: 3 0 3 8
key: 63 61 74
value: 07 07 07 07 07 07 07 07
end

# Maps after packet 3
nanotube_map: 0 0 3 8
key: 63 61 62
value: 03 03 03 03 03 03 03 03
key: 68 61 74
value: 02 02 02 02 02 02 02 02
key: 63 6f 64
value: 01 01 01 01 01 01 01 01
key: 66 6f 6f
value: 00 00 00 00 00 00 00 00
end

nanotube_map: 1 0 3 8
key: 63 61 74
value: 06 06 06 06 06 06 06 06
key: 77 69 70
value: 05 05 05 05 05 05 05 05
key: 62 61 72
value: 04 04 04 04 04 04 04 04
end

nanotube_map: 2 0 3 64
key: 72 65 73
value: 03 ff 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
end

nanotube_map: 3 0 3 8
key: 63 61 74
value: 07 07 07 07 07 07 07 07
end

# Testing kernel dag_switch_test
# Maps after packet 0
nanotube_map: 0 0 3 8
key: 63 61 62
value: 03 03 03 03 03 03 03 03
key: 68 61 74
value: 02 02 02 02 02 02 02 02
key: 63 6f 64
value: 01 01 01 01 01 01 01 01
key: 66 6f 6f
value: 00 00 00 00 00 00 00 00
end

nanotube_map: 1 0 3 8
key: 63 61 74
value: 06 06 06 06 06 06 06 06
key: 77 69 70
value: 05 05 05 05 05 05 05 05
key: 62 61 72
value: 04 04 04 04 04 04 04 04
end

nanotube_map: 2 0 3 64
key: 72 65 73
value: 03 ff 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 04 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
end

nanotube_map: 3 0 3 8
key: 63 61 74
value: 07 07 07 07 07 07 07 07
end

# Maps after packet 1
nanotube_map: 0 0 3 8
key: 63 61 62
value: 03 03 03 03 03 03 03 03
key: 68 61 74
value: 02 02 02 02 02 02 02 02
key: 63 6f 64
value: 01 01 01 01 01 01 01 01
key: 66 6f 6f
value: 00 00 00 00 00 00 00 00
end

nanotube_map: 1 0 3 8
key: 63 61 74
value: 06 06 06 06 06 06 06 06
key: 77 69 70
value: 05 05 05 05 05 05 05 05
key: 62 61 72
value: 04 04 04 04 04 04 04 04
end

nanotube_map: 2 0 3 64
key: 72 65 73
value: 03 ff 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 01 05 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 04 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
end

nanotube_map: 3 0 3 8
key: 63 61 74
value: 07 07 07 07 07 07 07 07
end

# Maps after packet 2
nanotube_map: 0 0 3 8
key: 63 61 62
value: 03 03 03 03 03 03 03 03
key: 68 61 74
value: 02 02 02 02 02 02 02 02
key: 63 6f 64
value: 01 01 01 01 01 01 01 01
key: 66 6f 6f
value: 00 00 00 00 00 00 00 00
end

nanotube_map: 1 0 3 8
key: 63 61 74
value: 06 06 06 06 06 06 06 06
key: 77 69 70
value: 05 05 05 05 05 05 05 05
key: 62 61 72
value: 04 04 04 04 04 04 04 04
end

nanotube_map: 2 0 3 64
key: 72 65 73
value: 03 ff 00 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 01 05 00 00 00 00 00 00 00
       02 07 00 00 00 00 00 00 00 00 00 04 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
end

nanotube_map: 3 0 3 8
key: 63 61 74
value: 07 07 07 07 07 07 07 07
end

# Maps after packet 3
nanotube_map: 0 0 3 8
key: 63 61 62
value: 03 03 03 03 03 03 03 03
key: 68 61 74
value: 02 02 02 02 02 02 02 02
key: 63 6f 64
value: 01 01 01 01 01 01 01 01
key: 66 6f 6f
value: 00 00 00 00 00 00 00 00
end

nanotube_map: 1 0 3 8
key: 63 61 74
value: 06 06 06 06 06 06 06 06
key: 77 69 70
value: 05 05 05 05 05 05 05 05
key: 62 61 72
value: 04 04 04 04 04 04 04 04
end

nanotube_map: 2 0 3 64
key: 72 65 73
value: 03 03 ff 00 00 00 00 00 00 00 00 00 00 00 00 00
       00 00 00 00 00 00 00 01 05 00 00 00 00 00 00 00
       02 07 00 00 00 00 00 00 00 00 00 04 00 00 00 00
       00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
end

nanotube_map: 3 0 3 8
key: 63 61 74
value: 07 07 07 07 07 07 07 07
end

